winui/BusyIndicator/Getting-Started.md

```diff
@@ -1,12 +1,114 @@
 ---
 layout: post
-title: Getting Started with WinUI BusyIndicator control | Syncfusion
-description: Learn all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control, its elements, and more here.
+title: Getting started with WinUI BusyIndicator control | Syncfusion
+description: Learn here all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control and its basic features.
```

Comment:
description: Learn all about getting started with the Syncfusion WinUI BusyIndicator(SfBusyIndicator) control and its basic features here.

Reply:
Changed the content description as mentioned above.